packages/jss-plugin-vendor-prefixer/src/index.js

```diff
@@ -15,12 +15,13 @@ export default function jssVendorPrefixer(): Plugin {
     }
   }
 
-  function onProcessStyle(style, rule) {
-    if (rule.type !== 'style') return style
-
+  function prefixStyle(style) {
     for (const prop in style) {
       const value = style[prop]
-
+      if (prop === 'fallbacks' && Array.isArray(value)) {
```

Comment:
Should we also handle fallbacks which are just an object?

Reply:
adding a test

Reply:
added a test, it already works.